targets/Zynq-Ultrascale+/Mars_XU3/Mars_ST3/build.ini

```diff
@@ -9,7 +9,7 @@ boot_files_2eg_1i = false
 [boot_files_3eg_2]
 unpack = true
 force_download = false
-url = http://www.enclustra.com/binaries/enclustra-bsp/{ebe_release}/refdes/mars_xu3_eb1_3eg_2i.zip
+url = http://www.enclustra.com/binaries/enclustra-bsp/{ebe_release}/refdes/mars_xu3_st3_3eg_2i.zip
```

Comment:
Should this be `2i`? The section name does not contain the `i` suffix. I guess either the binary or the whole section should be renamed so they match.

Reply:
It depends. I just adopted the existing. To be consistent with other "build.ini" files it seems we should get rid of the "i" on the end.

Reply:
You are right, it is quite inconsistent in other places too.

I guess it will be easier to update the section names rather than the binary names so we don't have to move the binaries on the server.

Let's ignore it for now though and fix it globally in the future. Thanks!